### PR TITLE
added pattern background to hero, footer sections

### PR DESCRIPTION
**Pull request by reveurguy**
Added background to hero,footer sections as discussed in #38921 

https://deploy-preview-38959--kubernetes-io-main-staging.netlify.app/

### CODE REVIEW

**Comment by NitishKumar06** [comment]
Please provide a preview link on the PR description itself.

**Comment by NitishKumar06** [comment]
Couple of things to add more:

1. Prefer working on that issue which has the tag `triage/accepted`
2. Before working on an issue with the tag `triage/accepted` , assign yourself by `/assign` to avoid conflicts.

File: assets/scss/_base.scss
Comment (NitishKumar06) [inline assets/scss/_base.scss:155]: The design looks good. Is there a way to keep the weight/ratio of the pages in sync with the current design present?
image

image

**Comment by reveurguy** [comment]
Shall I open a new issue regarding the background changes? We can keep #38921 for discussion purposes and I can create new issues like the card structure one for the PRs

**Review by NitishKumar06 — CHANGES_REQUESTED** [review]
A little suggestion above (if it works). However, we'd be willing for this to be reviewed by other maintainers before proceeding next.

**Comment by NitishKumar06** [comment]
> Shall I open a new issue regarding the background changes? We can keep #38921 for discussion purposes and I can create new issues like the card structure one for the PRs

I'd not prefer this. The suggestions will most likely be provided under the PR.

File: assets/scss/_base.scss
Comment (reveurguy) [reply assets/scss/_base.scss:155]: Sorry, i couldn't understand what you meant by weight/ratio of the pages.

**Comment by reveurguy** [comment]
> Couple of things to add more:
> 
> 1. Prefer working on the issue which has the tag `triage/accepted`
> 2. Before working on an issue with the tag `triage/accepted` , assign yourself by `/assign` to avoid conflicts.

ok yeah, can you add the triage/accepted to #38921 if there's no issue?

**Comment by sftim** [comment]
>  can you add the triage/accepted to https://github.com/kubernetes/website/issues/38921 if there's no issue?

We (SIG Docs) should only add that after discussing the proposal and agreeing that we want _all_ the changes that that issue is suggesting.

**Comment by reveurguy** [comment]
Ok yeah, understood

**Comment by reveurguy** [comment]
Any update on this?

File: assets/scss/_base.scss
Comment (NitishKumar06) [reply assets/scss/_base.scss:155]: In short, Is there a way to maintain the size ratio same as it was before?

Comment (reveurguy) [reply assets/scss/_base.scss:155]: @NitishKumar06 what size are you viewing the site in? as shown in the screenshots below, the size/ratio looks fine.

Screenshot 2023-01-25 at 12 42 27 AM

Screenshot 2023-01-25 at 12 42 37 AM

**Comment by kbhawkey** [comment]
👋 @reveurguy . Thanks for contributing.

I looked at the preview of the changes (adds more blue).
This type of change should be discussed at a docs meeting (and possibly communicated to a broader audience).
In the mean time, I'll hold this pull request.
/hold

**Comment by divya-mohan0209** [comment]
Hello @reveurguy , would you be able to add this to the agenda for our next SIG Docs meeting for further discussion? This has been pending for quite sometime now.

**Comment by divya-mohan0209** [comment]
Hello @reveurguy, due to lack of response/activity on this PR since quite some time, I will be closing off this PR. Thank you so much for suggesting this change & for your efforts in opening the PR. If you wish to continue working on the PR, I'd request you to 
- Add it to the agenda for any upcoming SIG Docs meeting & socialise the idea.
- Once there is consensus, reopen this PR & we can continue work on it.

/close

**Pull request closed without merging**
